README.md

```diff
@@ -1 +1,37 @@
 
+For this example ill be using Management Services, which is a logical group of services that are deployed on management hubs (Previously known as Management Hub Components).
+
+2 Repos will coexist:
+- Preprod Management Service Set
+- Prod Management Service Set
+
+ServiceSet - A repo template for Argo ApplicationSet which deploys a set of services defined in values.yaml to a specific cluster.
+
+A service is a helm chart reference defined by:
+- serviceName
+- repoURL
+- targetRevision
+- path
+
+Lets say I want to add a feature to a project that is a part of Management Service Set (After application CI has passed successfully).
+I'll create a new branch for the feature in the helm chart and make the desired changes.
```

Comment:
Maybe add some example to ensure that everything is more clear for the reader

Reply:
👍